translate/Main.hs

```diff
@@ -127,12 +134,26 @@ initVal s ((ls,v):l)
         | otherwise = initVal s l
 initVal _ _ = 0
 
+signalLists :: [DynSignal] -> [String]
+signalLists xs = map (\s -> show s) xs
```

Comment:
This is just `signalLists = map show` :) Since it's so short and easy to understand I suggest to use `map show` directly without using a helper function.